gui/symfony-docker/src/Controller/EleveurController.php

```diff
@@ -48,27 +63,38 @@ public function index(): Response
         return $this->render('pro/eleveur/index.html.twig');
     }
 
+
+
     #[Route('/naissance', name: 'app_eleveur_naissance')]
     public function naissance(Request $request,
                               ResourceHandler $handler): Response
     {
         $form = $this->createForm(EleveurBirthType::class,
-            $resource = $handler->createDefaultNewResource($this->getUser()));
-        $form->handleRequest($request);
-        if ($form->isSubmitted() && $form->isValid()) {
-            try {
-                $this->entityManager->persist($resource);
-                $this->entityManager->flush();
-            } catch (UniqueConstraintViolationException){
-                $this->addFlash('error', 'Le tag NFC est déjà utilisé');
-                return $this->redirectToRoute('app_eleveur_naissance');
-            }
-            $this->addFlash('success', 'La naissance de votre animal a bien été enregistrée !');
-            return $this->redirectToRoute('app_eleveur_index');
+        $resource = $handler->createDefaultNewResource($this->getUser()));
+    $form->handleRequest($request);
+    if ($form->isSubmitted() && $form->isValid()) {
+        try {
+            // dd($form->getData());
+            $metadata = $this->hardwareService->metadataTemplate((int)$form->getData()->getWeight(),
+                                                                (int)$form->getData()->getPrice(),
+                                                                $form->getData()->getDescription(),
+                                                                $form->getData()->getGenre()
+                                                            );
+            $response = $this->hardwareService->mintResource(1,1, ['metadata' => $metadata]);
```

Comment:
so here you will need to use the templates that are saved in the contract. you do not know what ressource is behind the nummero 1. 

what is needed: 
1. getTemplates that return the templates from the contract (implement on the BC24-API side)
2. filter those for role BREEDER
3. provide the user with a dropdown to select the animal he wants to create
4. from this selection you will be able to get the templateID